package.json

```diff
@@ -4,7 +4,7 @@
   "private": "private",
   "scripts": {
     "start": "node ./bin/www",
-    "test": "export NODE_ENV=test && mocha test"
+    "test": "NODE_ENV=test npm test"
```

Comment:
Additinally, you should rely on the mocha version installed for this project, concretely in `./node_modules/mocha/bin/mocha test`.

There is another subtle problem. You are relying on _dotenv-safe_, which means that you don't have any step where you instruct the user to export the required env variables.

To fix this problem you have to explicitly export the env vars in the `test` script.

```javascript
"test": "NODE_ENV=test TWILIO_ACCOUNT_SID=ACXXXXXXX ...."
```

For referencing _mocha_ please use `./node_modules/.bin/mocha`.